assets/js/modules/ads/components/setup/SetupMainPAX.js

```diff
@@ -146,6 +149,8 @@ export default function SetupMainPAX( { finishSetup } ) {
 			return;
 		}
 
+		await trackEvent( `${ viewContext }_pax`, 'pax_campaign_created' );
```

Comment:
We should only await `trackEvent` when we are invoking it before a navigation. This ensures that the event is tracked before JS execution is halted by navigation. For any other on page events, it's safe (and preferable) to not await it and let it happen async as it may otherwise introduce latency into the action which we don't want in actions where it isn't necessary as described.

Reply:
Fixed, thanks @aaemnnosttv